addon/serializers/application.js

```diff
@@ -86,7 +86,7 @@ export default DS.RESTSerializer.extend({
         // the links property so the adapter can async call the
         // relationship.
         // The adapter findHasMany has been overridden to make use of this.
-        if(options.relation) {
```

Comment:
[Release notes](http://emberjs.com/blog/2015/05/21/ember-data-1-0-beta-18-released.html) said that the typeForRoot function had been changed to modelNameFromPayloadKey